templates/zerver/app/index.html

```diff
@@ -147,6 +147,7 @@
 <div id="scheduled_messages_overlay_container"></div>
 <div id="settings_overlay_container" class="overlay" data-overlay="settings" aria-hidden="true">
 </div>
+<div id="message_edit_history_container"></div>
```

Comment:
Call this `message_edit_history_overlay_container` to align with existing values.

Reply:
Once the issue with the overlay container width (https://chat.zulip.org/#narrow/stream/101-design/topic/Redesign.20.22Message.20edit.20history.22.20modal.20.2328695/near/1744049) is resolved, I will update it in the next push.

Reply:
`message_edit_history_container` has been renamed to `message_edit_history_overlay_container`